Main/Source/nonhuman.cpp

```diff
@@ -922,6 +922,29 @@ truth spider::SpecialBiteEffect(character* Char, v2, int, int, truth BlockedByAr
     return false;
 }
 
+truth vampirebat::SpecialBiteEffect(character* Victim, v2 HitPos, int BodyPartIndex, int Direction, truth BlockedByArmour, truth Critical, int DoneDamage)
```

Comment:
I agree it could be optimized, but there are some subtle differences with vampire bats. For instance bites from a vampire bat accumulate the amount of vampirism, contributing to a potential permanent state. You have to get bitten a lot though. Normal vampires don't pass on the vampirism state to other characters who already have vampirism.
Plus as they are written now, vampire bats use their special bite slightly less often than vampires.